accname/manual/description_test_case_557-manual.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>Description test case 557</title>
-    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+  <meta charset="utf-8">
```

Comment:
correcting whitespace alignment mistake in the diff
```suggestion
    <meta charset="utf-8">
```